Add Text to PDF Converter Web App

Srushtee1706 opened the pull request:
@rutikakengal 
This PR adds a simple yet fully functional "Text to PDF Converter" web app using HTML, CSS, and JavaScript.

## Features:
- Textarea for user input
- Button to download input as PDF
- A4 format PDF using jsPDF
- Clean UI and responsive design

## Demo:
- Type text and click "Download PDF"
- PDF will be saved locally as `text-to-pdf.pdf`

Closes #201

rutikakengal commented:
Resolve the conflicts and Update the branch as new changes are made by the contributors   @Srushtee1706

Srushtee1706 commented:
hey @rutikakengal , conflicts have been solved please review it

rutikakengal commented:
See the deploy preview of ur work @Srushtee1706 
https://deploy-preview-230--100daysof100webprojects.netlify.app/

rutikakengal commented:
Why u r adding in middle of Days ! ur project must add at the ending at which Day is currently going on ! and see where another Projects links are not working ! 

Check this Deploy Preview here -->
https://deploy-preview-230--100daysof100webprojects.netlify.app/

Srushtee1706 commented:
hey @rutikakengal ,
i have added I have added my new project "Text to PDF Converter" as Project 18 (placed at the end of the list).
Also, I have fixed the issue where other project links were showing "Page Not Found"  all demo links are now working properly.
